be/src/exec/parquet_reader.cpp

```diff
@@ -204,13 +208,23 @@ Status ParquetReaderWrap::read(Tuple* tuple, const std::vector<SlotDescriptor*>&
     uint8_t tmp_buf[128] = {0};
     int32_t wbytes = 0;
     const uint8_t *value = nullptr;
+    int index = 0;
     int column_index = 0;
     try {
         size_t slots = tuple_slot_descs.size();
         for (size_t i = 0; i < slots; ++i) {
             auto slot_desc = tuple_slot_descs[i];
-            column_index = i;// column index in batch record
-            switch (_parquet_column_type[i]) {
+            auto iter = _columns_from_path.find(slot_desc->col_name());
```

Comment:
I think we can finish it in BrokerScanner's line_to_src_tuple. Which can avoid same code in different places.

Reply:
Currently, we have two broker scanners. One is BrokerScanner using for reading CSV like row-storage sources. The other is ParquetScanner using for read parquet files, which is a column-storage source. It seems that we cannot reuse the `line_to_src_tuple` method of BrokerScanner

Reply:
I think we can write a common function to fill partition columns which can be used in both two scanners, like `fill_dest_tuple`. If this function is not easy to write and used by scanners, it's OK to write one copy for each.